src/spikeinterface/extractors/neoextractors/plexon.py

```diff
@@ -15,22 +15,41 @@ class PlexonRecordingExtractor(NeoBaseRecordingExtractor):
 
     Parameters
     ----------
-    file_path : str
+    file_path : str | Path
         The file path to load the recordings from.
     stream_id : str, default: None
         If there are several streams, specify the stream id you want to load.
     stream_name : str, default: None
         If there are several streams, specify the stream name you want to load.
     all_annotations : bool, default: False
         Load exhaustively all annotations from neo.
+    use_names_as_ids : bool, default: True
+        If True, the names of the signals are used as channel ids. If False, the channels ids are the ids provided
+        by the NeoRawIO.
+
+        Example for wideband signals:
+            names: ["WB01", "WB02", "WB03", "WB04"]
+            ids: ["1" , "2", "3", "4"]
```

Comment:
```suggestion
            ids: ["0" , "1", "2", "3"]
```

seems like we start the ids from '0', just so we are accurate :)

Reply:
Yes, my mistake.